ResearchKit/Common/ORKHelpers.h

```diff
@@ -34,6 +35,37 @@
 #import <ResearchKit/ORKErrors.h>
 
 
+#if ( defined(ORK_LOG_LEVEL_NONE) && ORK_LOG_LEVEL_NONE )
+#  undef ORK_LOG_LEVEL_DEBUG
+#  undef ORK_LOG_LEVEL_WARNING
+#  undef ORK_LOG_LEVEL_ERROR
+#endif
+
+#if ( !defined(ORK_LOG_LEVEL_NONE) && !defined(ORK_LOG_LEVEL_DEBUG) && !defined(ORK_LOG_LEVEL_WARNING) && !defined(ORK_LOG_LEVEL_ERROR) )
+#  define ORK_LOG_LEVEL_WARNING 1
```

Comment:
Thanks, use `ORK_LOG_LEVEL_WARNING` as the default is good. 
Shall we update RK's project file to include our recommended configurations: 

Debug | ORK_LOG_LEVEL_WARNING=1
Release | ORK_LOG_LEVEL_NONE=1